lib/actions/user.js

```diff
@@ -181,3 +188,91 @@ export function deleteUserMonitoredTrip (id) {
     }
   }
 }
+
+/**
+ * Requests a verification code via SMS for the logged-in user.
+ */
+export function requestPhoneVerificationSms (originalUserData, newPhoneNumber) {
+  return async function (dispatch, getState) {
+    const { otp, user } = getState()
+    const { otp_middleware: otpMiddleware = null } = otp.config.persistence
+
+    if (otpMiddleware) {
```

Comment:
Good suggestion for throwing an error if `otpMiddleware` is null.

Reply:
I have been intending to refactor the code involving the extraction and use of the `otpMiddleware` variable, it would be best to do that in a separate PR.

Reply:
Addressed separately in in https://github.com/opentripplanner/otp-react-redux/pull/246.